pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -283228,7 +283228,6 @@ self: {
        description = "A modern testing framework for Haskell with good defaults and advanced testing features";
        license = "unknown";
        hydraPlatforms = lib.platforms.none;
```

Comment:
You'll also need to remove this line as well, otherwise Hydra won't build the package.

```suggestion
```